script/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Scripts
+
+This folder is intended for various scripts used during the OTP development. They are provided 
+"as is" and the "owner" may do whatever she/he likes with it. 
+
+If you want to submit your own scripts, you need to include:
+ - A header at the beginning of the script stating who the owner is.
+ - The scrip should print some usage documentation if invoked with `--help` and `-h`.
```

Comment:
```suggestion
 - The script should print some usage documentation if invoked with `--help` and `-h`.
```

Reply:
Should add that doing proper long-option parsing is not possible with the standard `getopts` command.